sleap/gui/commands.py

```
@@ -3470,13 +3473,293 @@
+        else:
+            params["frames"] = range(context.state["video"].frames)
+
+        params["show_edges"] = context.state.get("show edges", default=True)
+        params["edge_is_wedge"] = (
+            context.state.get("edge style", default="").lower() == "wedge"
+        )
+        params["marker_size"] = context.state.get("marker size", default=4)
+
+        return True
+    
+class ExportVideoClip(AppCommand):
```

Ruff (0.8.2)

Redefinition of unused `ExportVideoClip` from line 3476

(F811)